### PR TITLE
Feature/onboarding

### PR DESCRIPTION
**Pull request by Zempoalteca**
## Description
Update onboarding
- Change Images
- Update Colors
- Update texts
- Change Loading view for register Country process

### CODE REVIEW

File: sosmex/Modules/Countries.swift [hunk 1]
```diff
@@ -24,11 +24,11 @@ let colombia = (name: "Colombia", code: 57,
 let chile = (name: "Chile", code: 56,
     region: "CL", imageName: "ic_flag_chile")
 //Peru
-let peru = (name: "Peru", code: 51,
+let peru = (name: "Perú", code: 51,
```
Comment (oscarito9410) [inline sosmex/Modules/Countries.swift:27]: good catch

File: sosmex/Modules/Countries.swift [hunk 2]
```diff
@@ -24,11 +24,11 @@ let colombia = (name: "Colombia", code: 57,
...
     region: "PE", imageName: "ic_flag_peru")
 //Usa
 let usa = (name: "Estados Unidos", code: 1,
     region: "US", imageName: "ic_flag_usa")
 //Canada
-let canada = (name: "Canada", code: 1,
+let canada = (name: "Canadá", code: 1,
```
Comment (oscarito9410) [inline sosmex/Modules/Countries.swift:33]: same as above

File: sosmex/Modules/OnboardingPageViewController .swift [hunk 3]
```diff
@@ -28,39 +31,40 @@ class OnboardingPageViewController: UIPageViewController {
             for index in 0 ..< onboardingItems.count {
                 items.append(self.instanceViewController(index: index))
             }
-            items.insert(instanceCountryViewController(), at: positionCountryPicker)
+            if FeatureConfigImpl.isCountryChooseEnabled {
+                items.insert(instanceCountryViewController(), at: positionCountryPicker)
```
Comment (oscarito9410) [inline sosmex/Modules/OnboardingPageViewController .swift:35]: cool

**Review by oscarito9410 — DISMISSED** [review]
outstanding work!

**Comment by oscarito9410** [comment]
outstanding work!!

File: sosmex/Modules/OnboardingViewController.swift [hunk 4]
```diff
@@ -47,7 +46,9 @@ class OnboardingViewController: UIViewController {
     }
 
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        guard segue.identifier == segueName, let destination = segue.destination as? OnboardingPageViewController else {
+        guard segue.identifier == segueName,
+            let destination = segue.destination as? OnboardingPageViewController else {
+
```
Comment (alexhelyer) [inline sosmex/Modules/OnboardingViewController.swift:51]: `if let` is not bad idea. It's less source code.

**Review by alexhelyer — APPROVED** [review]
Great job!

File: sosmex/Modules/OnboardingViewController.swift (hunk 4 above)
Comment (Zempoalteca) [reply sosmex/Modules/OnboardingViewController.swift:51]: You're right! I will do it.

**Review by alexhelyer — APPROVED** [review]

**Pull request merged**